tests/test_whiteblacklist

```diff
@@ -140,6 +140,7 @@ if [ -e "$BLACKLIST_FILE" ]; then
     mv $BLACKLIST_FILE $BLACKLIST_FILE_BAK
 fi
 
+echo "Starting white / black list test. Make sure you have crash report autoupload disabled"
```

Comment:
Can we check autoupload status and refuse to continue if enabled? Instead of leaving it up to user to spot the warning. And isn't it sort of already too late here?

Reply:
Everything is possible, but that would require reading the value from configuration file in a shell script, and the configuration is maybe somewhat private to the crash-reporter anyway. I think this quite rarely gets executed by any developer so wouldn't spend too much time on that.